### ISSUE TITLE
Include infinite/"radio style" playlist

### ISSUE THREAD

**Issue by Mabenan** [mopidy/mopidy-spotify#329]
I known that this is a a complex one as it isnt solved with a simple api call neither by spotify nor by mopidy. But i would really like to have it. So this is will be more of a discussion thread how it should behave and how we can integrate it.

The talk is about the spotify (or any other big music streaming service) auto play. Which is basicaly a never ending queue of tracks based on the tracks already listen to in the session.

Spotify has a recommandation api which we can maybe use to find next tracks

**Comment by Mabenan** [comment]
I dont think we need a timer for this. I think it would be enough to determine the next tracks when the tracklist has come to the last or the last 2 tracks. 

So the sxample of behaviour would be:

User adds tracks to a tracklist lets say 5 tracks.
The music is playing and when the playing reach the last track. The new feauture will automatically add 5 or so tracks to the tracklist.

Im not familiar with the life cycles in mopidy but when it is possible to listen on the track changes of the playback we could implement something that checks on each track change if we reached the end of the tracklist and add the next track.

Of course this feature would come with limitations:
The random funktion would not work as it means the tracklist would falsly reach the end of the tracklist.

**Comment by kingosticks** [comment]
Someone could add this as a feature. Spotify artist radio is like this, in theory librespot should support it, if it still exists...? But We would need something that waited for the end of track event and then fetched a load more tracks to play if it was the last one. I'm not sure if we have everything available to do that but someone who is interested could investigate.

**Comment by kingosticks** [comment]
Also, I'm going to rename this because I consider "autoplay" functionality to play something automatically on startup. This isn't that feature.
